### DISCUSSION TITLE
add a default project for new OIDCuser ?

### DISCUSSION THREAD

**Discussion by ydastous** [semaphoreui/semaphore#2289] [Ideas]
When a user sign in for the first time via OIDC, He can not do anything until the admin put the user in a team, It will be great if we can assigned a default project for new user. like a config default.

**Comment by lafayetteduarte** [comment]
If I may expand on your idea .

The config could algo allow for mapping a claim / role to a team / is admin on semaphore.

eg: if the token contains a admin role , this person is admin on semaphore.

If the token contains a claim , the user gains access to a specific team .
